### PR TITLE
[REVIEW] Feature adding local logging dirrectory

### LINKED ISSUE #1083: [FEA] Optionally setting a Blazing local logging directory
**Is your feature request related to a problem? Please describe.**
If we run a script with a remote dask client attached to a Blazing Context and we pass a path to the  BLAZING_LOGGING_DIRECTORY config option. An error will raised if the folder passed don't exists on the client machine. A workaround consists of creating the same folder locally. But, it will depend on whether we have the required privileges to make it.

**Describe the solution you'd like**
If we are not able to create a folder, let's say because we lack privileges. We should pass another config option, for example, BLAZING_LOCAL_LOGGING_DIRECTORY pointing to a folder locally accessible.

### PR DESCRIPTION
**Pull request by rommelDB**
Fixes #1083 and #1106

### CODE REVIEW

File: tests/BlazingSQLTest/EndToEndTests/loggingTest.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    def executionTest():
+        # Run Query ------------------------------------------------------
+
+        print("==============================")
+        print(queryType)
+        print("==============================")
+
+        queryId = "TEST_01"
+        query = """select * from bsql_logs"""
```
Comment (williamBlazing) [inline tests/BlazingSQLTest/EndToEndTests/loggingTest.py:22]: Please change this query by adding a filter. After running thousands of queries, the return of this select * could be enormous.

**Review by williamBlazing — CHANGES_REQUESTED** [review]

File: tests/BlazingSQLTest/EndToEndTests/loggingTest.py (hunk 1 above)
Comment (rommelDB) [reply tests/BlazingSQLTest/EndToEndTests/loggingTest.py]: Fixed. Thanks.

**Comment by williamBlazing** [comment]
This also fixes #1106

**Comment by williamBlazing** [comment]
rerun-tests

**Pull request merged**